tests/integration.rs

```diff
@@ -358,6 +358,22 @@ fn sparse3(#[case] harness: Harness) {
 	assert_eq!(buf, expected);
 }
 
+/// This test checks, if the `file` utility crashes the filesystem or not.
```

Comment:
Rather than rely on `file`'s behavior, which may change, I think it would be a better regression test to just do the operation that causes the crash.  In this case, the operation that crashes the server is a 128k read at byte offset 549883084800.

Also, if this is related to a particular issue number, I suggest you put that in a comment by the test.